addons/functions/functions/fnc_decayPFH.sqf

```diff
@@ -67,16 +67,7 @@ if (isNil QGVAR(decayPFH)) then {
 						} else {
 							_trench setVariable ["ace_trenches_progress", _progress];
```

Comment:
as far as I can tell, here is the only instance where its not set globally. intentional?

Reply:
i have no idea how decay works but progress should never be local.